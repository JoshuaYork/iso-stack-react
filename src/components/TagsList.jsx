import React from 'react';
import { Link } from "react-router-dom";

export default ({tags})=> (
    <div>
        {tags.map(tag=><Link key={tag} to={`/tag/${tag}`}><button>{tag}</button></Link>)}
    </div>

)

