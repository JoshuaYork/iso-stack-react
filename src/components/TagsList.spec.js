import React from 'react';
import TagsList from './TagsList';
import renderer from 'react-test-renderer';

describe("The tags List", () => {
            it("renders as expected", () => {
                    const tree = renderer.create( < TagsList tags = {
                            ['css', 'html', 'go']
                        }
                        />).toJSON();

                        console.log(tree);
                        expect(tree).toMatchSnapshot();
                    });
            });