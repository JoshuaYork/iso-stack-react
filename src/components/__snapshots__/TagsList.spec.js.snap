// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The tags List renders as expected 1`] = `
<div>
  <code>
    <a
      href="/tags/css"
    >
      css
    </a>
  </code>
  <code>
    <a
      href="/tags/html"
    >
      html
    </a>
  </code>
  <code>
    <a
      href="/tags/go"
    >
      go
    </a>
  </code>
</div>
`;
